README.md

```diff
@@ -151,24 +155,24 @@ The react app in this quickstart shows us an example of the username/password au
 
 Sometimes, you would want to upload some files to the cloud. This can range from a profile pic for your user or images for things listed on your app. You can securely add, remove, manage, update files such as pictures, videos, documents using Hasura filestore.
 
-You can try out these in the `API EXPLORER` tab of the `api console`. To learn more, check out our [docs](https://docs.hasura-stg.hasura-app.io/0.15/manual/users/index.html)
+You can try out these in the `API EXPLORER` tab of the `api console`. To learn more, check out our [docs](https://docs.hasura-stg.hasura-app.io/0.15/manual/files/index.html)
```

Comment:
The previous link was correct.

Reply:
The previous link was point to "users and authentication" section which is not relevant to "files" and the new link points to "files" section where the reference is given on how to upload, download, delete, etc... That's the reason I made the above change.

Reply:
No, the link was reference for the usage of `api console`, which is present in the `users/index.html`.
How about including both links?